tasks.py

```diff
@@ -48,9 +48,9 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_dev_example": {
-            "nautobot_ver": "2.0.0",
+            "nautobot_ver": "2.3.0",
             "project_name": "nautobot-dev-example",
-            "python_ver": "3.11",
+            "python_ver": "3.12",
```

Comment:
I have a minor hesitation with this change. The pylint-nautobot currently does not support 3.12, and some of our apps depend on other apps that may not have this just yet. I think it is probably safest to keep this at 3.11 and follow up after getting all the other apps updated.

Reply:
Should we go with python 3.11 and nautobot 2.3.0 in the dev environment? There's nothing in python 3.12 that we need.

Reply:
Sounds reasonable to me, with the same comment as above for putting in 2.3.1 instead.